docs/phylum_analyze.md

```diff
@@ -17,6 +17,9 @@ phylum analyze [options] <lockfile>
 `--filter <filter>`
 &emsp; Provide a filter used to limit the issues displayed
 
+`-g`, `--group <group_name>`
+&emsp; Group to use for analysis
```

Comment:
All the other options for this command are documented as actions, but this one is a thing.

Reply:
I just copied what the CLI help had, so if we want to change it here we should change it there as well